src/qtism/runtime/tests/Route.php

```diff
@@ -1117,10 +1117,11 @@ public function branch($identifier): void
 
             if ($targetRouteItems[$occurence]->getTestPart() !== $this->current()->getTestPart()) {
                 // From IMS QTI:
-                // In case of an item or section, the target must refer to an item or section
-                // in the same testPart [...]
-                $msg = 'Branchings to items outside of the current testPart is forbidden by the QTI 2.1 specification.';
-                throw new OutOfBoundsException($msg);
```

Comment:
Yes. But instead of throwing an exception, we just ignore such branch rules and going to the next item (like there are no invalid branch rules).

https://oat-sa.atlassian.net/browse/TR-5760 - ACs 9-14